scripts/variantstore/wdl/GvsCreateFilterSet.wdl

```diff
@@ -293,7 +293,7 @@ task ExtractFilterTask {
   >>>
 
   runtime {
-    docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_07_14"
+    docker: "us.gcr.io/broad-dsde-methods/broad-gatk-snapshots:varstore_2023_07_17"
```

Comment:
Why did you need to update this docker? It looks like only the python (`variantstore`) one had code changes

Reply:
to ensure that `bulk_ingest_staging`, the staging branch, is keeping up-to-date with java changes in `ah_var_store`